### DISCUSSION TITLE
Can I add colors to bookmarks

### DISCUSSION THREAD

**Discussion by Donkey1022** [alefragnani/vscode-bookmarks#668] [Q&A] [answered]
Can I add colors to bookmarks

**Comment by alefragnani** [comment, marked as answer]
Hi @Donkey1022 ,

No, there is no support for colored bookmarks for now. However, there is a feature request for this in #243, but no ETA. 

I'm planning to create a roadmap for 2024, reviewing the open requests, based on their relevance and complexity. Feel free to upvote the item, so I'll have a better measure of how impactful it should be. 

Hope this helps
